code/lib/blocks/src/components/ArgsTable/ArgsTable.tsx

```diff
@@ -67,7 +67,7 @@ export const TableWrapper = styled.table<{
       },
 
       'th:last-of-type, td:last-of-type': {
```

Comment:
This applies globally to the `td` which might come back to bite us. 

I think it'd be safer to apply negative margin to the `StyledIconButton` that way we're scoping the positioning to only the element in question

Reply:
Good call. On it.